middleware/integration_test/docker-compose.yml

```diff
@@ -0,0 +1,135 @@
+version: "3"
+services:
+  bitcoind:
+    image: nicolasdorier/docker-bitcoin:0.17.0
+    networks:
+      local:
+        ipv4_address: 10.10.0.10
+    environment:
+      EXPOSE_TCP: "true"
+      BITCOIN_EXTRA_ARGS: |
+        regtest=1
+        whitelist=0.0.0.0/0
+        server=1
+        regtest.rpcallowip=10.211.0.0/16
+        regtest.rpcallowip=172.17.0.0/16
+        regtest.rpcallowip=192.168.0.0/16
+        regtest.rpcallowip=0.0.0.0/16
+        regtest.rpcbind=0.0.0.0
+        rpcport=18443
+        rpcuser=rpcuser
+        rpcpassword=rpcpass
+    expose:
+      - "18443"
+      - "18444"
+    ports:
+      - "0.0.0.0:18443:18443"
```

Comment:
generally no need for expose in docker-compose (the ports you need should already be exposed in the Dockerfile)
then you can just use:
```
ports:
      - 18443:18443
      - 18444:18444
```

also without the full ip